foca/models/config.py

```diff
@@ -600,11 +620,14 @@ class LogFormatterConfig(FOCABaseConfig):
     """
     class_formatter: str = Field(
         "logging.Formatter",
-        alias="class",
+        env="CLASS",
     )
     style: str = "{"
     format: str = "[{asctime}: {levelname:<8}] {message} [{name}]"
 
+    class Config:
+        env_prefix = 'LOG_'
```

Comment:
This should be able to be set dynamically, either by passing a param to Config (if possible) or via an env variable (perhaps better). I think it would be quite important in order to namespace a given app